static/css/main.css

```diff
@@ -31,3 +31,21 @@ ul {
   margin: 0;
   padding-left: 1em;
 }
+
+@media only screen and (max-width : 450px) {
+  .org {
+    flex: 100%;
+  }
+}
+
+@media only screen and (min-width : 450px) and (max-width : 600px) {
+  .org {
+    flex: 50%;
+  }
+}
+
+@media only screen and (min-width : 600px) and (max-width : 750px) {
+  .org {
+    flex: 33%;
```

Comment:
This doesnt add up to 100% ..

Reply:
what? this is for 3 column

Reply:
Yeah, but 33 * 3 is not 100

Check out bootstrap grid
https://github.com/twbs/bootstrap/blob/v4-dev/dist/css/bootstrap-grid.css#L128-L131